doc/source/whatsnew/v0.21.0.txt

```diff
@@ -43,9 +43,10 @@ Backwards incompatible API changes
 
 - Support has been dropped for Python 3.4 (:issue:`15251`)
 - The Categorical constructor no longer accepts a scalar for the ``categories`` keyword. (:issue:`16022`)
-
 - Accessing a non-existent attribute on a closed :class:`HDFStore` will now
   raise an ``AttributeError`` rather than a ``ClosedFileError`` (:issue:`16301`)
+- Compression defaults in HDF stores now follow pytable standards. Default type is 'zlib' and default level is 0 (:issue:`15943`)
```

Comment:
yes, complevel=0 turns off all compression and the files are identical byte-for-byte (except possibly for the [object id's](http://www.pytables.org/usersguide/libref/hierarchy_classes.html#tables.Leaf.object_id) that are assigned to the the leafs in the file and may change from run to run).

I think maybe the whole problem is that we talk about what the defaults are, not what they do? Maybe rewrite the whatsnew entry so that it says:
> Compression defaults in HDF stores now follow pytable standards. Default is no compression and if complib is missing and complevel > 0 zlib is used.